index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Page Title</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<link rel="stylesheet" href="css/style.css" />
+	<base target="_blank">
+</head>
+<body>
+	<div class="container">
+		<div class="header">
```

Comment:
If you need 3 column layout it better to use columns with `width: 33%` or `display: flex` for container of columns.